code/game/objects/items/RCD.dm

```diff
@@ -1106,7 +1106,14 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 	var/current_layer = "Default Layer"
 	///Current selected color, for ducts
 	var/current_color = "omni"
-
+	///maps bit flag to layer number value. didnt make this global cause only this class needs it
+	var/static/list/name_to_number = list(
+		"First Layer" = 1,
```

Comment:
Use the layer name defines directly

Reply:
where are the defines located?

Reply:
code/__DEFINES/plumbing.dm

```
GLOBAL_LIST_INIT(plumbing_layers, list(
	"First Layer" = FIRST_DUCT_LAYER,
	"Second Layer" = SECOND_DUCT_LAYER,
	"Default Layer" = THIRD_DUCT_LAYER,
	"Fourth Layer" = FOURTH_DUCT_LAYER,
	"Fifth Layer" = FIFTH_DUCT_LAYER,
))
```
They used the string directly just like me. They only defined numerical bit flags ,not string for the layers

Reply:
Oh I thought these were already defined, nevermind the code for this is already weird and someone else would have to fix it anyway

Reply:
I'll mark this is as resolved cause i don't want to make global defines in this file which will only be used by my code anyway